rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] Kindly add your person and book variables to attribute accessor, based on the requirements, `Getters and Setters for @book and @person instance variables should be present`

Reply:
Well resolved ✔️